renovate.json

```diff
@@ -2,16 +2,25 @@
   "extends": [
     "config:base"
   ],
-  "labels": [
-    "kokoro:run"
-  ],
-   "packageRules": [
+  "packageRules": [
+    {
+      "packagePatterns": [ "^com.google.appengine:" ],
+      "groupName": "AppEngine packages"
+    },
     {
-      "packagePatterns": ["^io.grpc:grpc-"],
+      "packagePatterns": [ "^io.grpc:grpc-" ],
       "groupName": "gRPC packages"
+    },
+    {
+      "packagePatterns": [ "^io.vertx:vertex-" ],
+      "groupName": "Vertx packages"
+    },
+    {
+      "packagePatterns": [ "^io.ktor:ktor-" ],
+      "groupName": "Ktor packages"
```

Comment:
An option for `ktor` is to import the [`io.ktor:ktor-bom`](https://search.maven.org/artifact/io.ktor/ktor-bom) in a single place which should include all of the ktor artifacts.

Reply:
@averikitsch FYI (I believe these dependencies are in `appengine-java11/`)

Reply:
Can you add micronaut as well.

Reply:
The [only instance of micronaut I can find is already using the bom](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/e95a9ede3bf45bc70f17fd4279bb431c64af745d/appengine-java11/micronaut-helloworld/pom.xml#L41).

Should I remove ktor from here and just change to the bom instead?

Reply:
I've updated Ktor to use the bom PR #1915

Reply:
Updated this PR.